src/pharmpy/plugins/nlmixr/error_model.py

```diff
@@ -0,0 +1,393 @@
+from typing import Dict, Tuple, Union
+
+import pharmpy.model
+from pharmpy.deps import sympy
+from pharmpy.internals.expr.subs import subs
+
+from .CodeGenerator import CodeGenerator
+from .sanity_checks import print_warning
+
+
+def find_term(
+    model: pharmpy.model.Model, expr: sympy.Add
+) -> Tuple[Union[sympy.Symbol, sympy.Add], Dict]:
+    """
+    For a given expression for the dependent variable, find the terms
+    connected to the actual result and the terms connected to the error model.
+
+    Parameters
+    ----------
+    model : pharmpy.model
+        A pharmpy model object
+    expr : sympy.Add
+        An expression for the dependent variable. Should be a sympy.Add statement
+
+    Raises
+    ------
+    ValueError
+        If the model either has multiple additative- or proportional error
+        terms, the function will raise a ValueError
+
+    Returns
+    -------
+    res : Union[sympy.Symbol, sympy.Add]
```

Comment:
I am not sure this is the correct way to document returning a tuple, but I might be wrong.

Reply:
This was the style of the automatically generated docstring, which I chose to use. If you'd like me to change it, please let me know! Otherwise, it will remain the same.

Reply:
Ok, let's keep it. Since it is not the top level API docstring it is not super important